build_tools/requirements.txt

```diff
@@ -29,3 +29,7 @@ matplotlib==3.4.3; sys_platform == 'darwin'
 lxml
 pytools 
 cffi
+dominate
+html5lib
+importlib-resources
+importlib_resources
```

Comment:
Actually, 3.6 is EOL, so we should probably stop supporting it. However, this is a larger scope decision so I agree with the double import at present

Reply:
OK